### PR TITLE
fix: mf-5994 check if ValueRef is ready

### PR DESCRIPTION
**Pull request by UncleBill**
https://mask.atlassian.net/browse/MF-5994

### CODE REVIEW

File: packages/plugin-infra/src/manager/site-adaptor.ts [hunk 1]
```diff
@@ -34,7 +34,16 @@ useActivatedPluginsSiteAdaptor.visibility = {
 }
 
 export function useIsMinimalMode(pluginID: string) {
-    return useValueRef(minimalModeSub).includes(pluginID)
```
Comment (Jack-Works) [inline packages/plugin-infra/src/manager/site-adaptor.ts]: why do you need this change? useValueRef already handles `readyPromise`.

Comment (UncleBill) [reply packages/plugin-infra/src/manager/site-adaptor.ts]: The result was `false` by default, that's arbitrary.

Comment (UncleBill) [reply packages/plugin-infra/src/manager/site-adaptor.ts]: @Jack-Works Any other questions?

Comment (Jack-Works) [reply packages/plugin-infra/src/manager/site-adaptor.ts]: the `use(ref.readyPromise)` should suspend the render, you should not get a "default" value before the `readyPromise` is resolved.

Comment (UncleBill) [reply packages/plugin-infra/src/manager/site-adaptor.ts]:
```ts
        if (!ref2.nowReady) use(ref2.readyPromise)
```
Should be 
```ts
        if (!ref2.ready) use(ref2.readyPromise)
```

Comment (Jack-Works) [reply packages/plugin-infra/src/manager/site-adaptor.ts]: nice catch

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**